add-on/chrome/bootstrap.js

```diff
@@ -469,6 +469,11 @@ var WindowListener = {
           // Do the normal behavior first.
           controller._doCommand.apply(controller, arguments);
 
+          // Don't bother prompting the user if already sharing.
+          if (this.MozLoopService.screenShareActive) {
```

Comment:
to fullfill all the conditions, I think we should check also that:
- `loop.copy.ticket != -1`, that shows that Ctrl-C has been used before
- `roomList.length != 0`, that shows that a call has been done before (no matter if currently ongoing)